course_link.py

```diff
@@ -26,8 +26,7 @@ def main():
             titre = document['clients'][0]['output']['parameters'][0]['parameters'][5]['mapping'][name]
         except KeyError:
             titre = name
+    print(titre)
```

Comment:
En ajoutant `from sys import platform` au début, et:
```suggestion
    if platform == "linux" or platform == "linux2":
        print(titre.encode('utf-8'))
    else:
        print(titre)
```
?